runtime/near-vm-runner/src/tests/cache.rs

```diff
@@ -95,19 +95,30 @@ fn test_wasmer2_artifact_output_stability() {
     //
     // Note that this test is a best-effort fish net. Some changes that should modify the hash will
     // fall through the cracks here, but hopefully it should catch most of the fish just fine.
-    let mut hasher = StableHasher::new();
-    let code = near_test_contracts::trivial_contract();
-    let config = VMConfig::test();
-    let prepared_code = prepare::prepare_contract(code, &config).unwrap();
-    let mut features = CpuFeature::set();
-    features.insert(CpuFeature::AVX);
-    let triple = "x86_64-unknown-linux-gnu".parse().unwrap();
-    let target = Target::new(triple, features);
-    let vm = Wasmer2VM::new_for_target(config, target);
-    let artifact = vm.compile_uncached(&prepared_code).unwrap();
-    let serialized = artifact.artifact().serialize().unwrap();
-    serialized.hash(&mut hasher);
-    assert_eq!(hasher.finish(), 18352148442716835594, "WASMER2_CONFIG needs version change");
+    let expected_hashes =
+        [18352148442716835594, 9541580911438661949, 1877160384252030064, 1939770487874739745];
+    let contracts = [
+        near_test_contracts::trivial_contract(),
+        near_test_contracts::rs_contract(),
+        near_test_contracts::ts_contract(),
+        near_test_contracts::fuzzing_contract(),
```

Comment:
Hm, I think these three are expected to change somewhat frequently, and fixing the hashes can be a touch annoying. Even something like a rustc upgrade may change hashes, as we compile the contracts on the fly. 

I wonder if there's some smarter way to do this? For example, can we use wasm_smith's arbitrary to generate a deterministic, large contract?

Reply:
A good idea, I'll look into it.